File: Python Word Examples/ParagraphsApiDemo/deleteParagraph.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.ParagraphsApi(configuration)

file_name = "Template.docx"
node_path = "Section/0/Body/0"
index = 2
extra = {'folder': 'input', 'dest_file_path': 'output/DeleteParagraph.docx'}
result = api.delete_paragraph(file_name, node_path, index, **extra)

File: JavaScript Word Examples/ParagraphsApiDemo/deleteParagraph.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.ParagraphsApi(apiClient);

    var filename = "DeleteParagraph.docx";
    var nodePath = "Section/0/Body/0";
    var opt = {
        folder: 'input',
    };
    var index = 1;
    var destFilePath = "output/DeleteParagraph_output.docx";
    instance.deleteParagraph(filename, nodePath, index, destFilePath, opt, function (error, data, response) {
        if (error) throw error;
        });
})();